src/Xrefcheck/Verify.hs

```diff
@@ -71,6 +71,7 @@ import URI.ByteString qualified as URIBS
 
 import Xrefcheck.Config
 import Xrefcheck.Core
+import Xrefcheck.Data.Redirect
```

Comment:
Minor suggestion: since this is used only as part of config, I would re-export this in `Xrefcheck.Config`.

You can add an entire module re-export by placing e.g.

```
  ( -- export list
  , module Xrefcheck.Data.Redirect  -- this module should be imported too 
  , ... 
```